test/spectests/spectests.cpp

```diff
@@ -404,16 +422,22 @@ class test_runner
 
     std::optional<fizzy::Value> read_value(const json& v)
     {
-        const auto arg_type = v.at("type").get<std::string>();
-        if (arg_type != "i32" && arg_type != "i64" && arg_type != "f32" && arg_type != "f64")
+        const auto type = v.at("type").get<std::string>();
+
+        // JSON tests have all values including floats serialized as 64-bit unsigned integers
```

Comment:
Not that JSON supports the whole 64-bit 😉

Reply:
The format supports arbitrary long literals, in theory.